presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -343,14 +341,24 @@ private static boolean shouldUseFileSplitsFromInputFormat(InputFormat<?, ?> inpu
         }
         catch (NestedDirectoryNotAllowedException e) {
             // Fail here to be on the safe side. This seems to be the same as what Hive does
-            throw new PrestoException(HIVE_INVALID_BUCKET_FILES, format("%s Found sub-directory in bucket directory for partition: %s", CORRUPT_BUCKETING, splitFactory.getPartitionName()));
+            throw new PrestoException(
+                    HIVE_INVALID_BUCKET_FILES,
+                    format("Hive table %s.%s is corrupt. The files do not match the bucketing declaration. Found sub-directory in bucket directory for partition: %s",
```

Comment:
Remove the “files do not match” sentence since it’s not relevant

Reply:
done